src/lib/crawl.js

```diff
@@ -6,7 +6,7 @@ import request from 'request';
 import cheerio from 'cheerio';
 import * as async from 'async';
 import * as youtube from './youtube.js';
-import * as www from '../bin/www.js';
+import clientSecrets from '../background/init.js';
```

Comment:
We have now multiple exports at init.js so:
`-import clientSecrets from '../background/init.js';`
`+import {clientSecrets} from '../background/init.js';`